staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -260,9 +263,15 @@ func (c *Config) ApplySecureServingOptions(secureServing *options.SecureServingO
 		return c, nil
 	}
 
+	ipMax := c.MaxRequestsInFlight / 2
+	if ipMax > c.MaxMutatingRequestsInFlight/2 {
+		ipMax = c.MaxMutatingRequestsInFlight / 2
+	}
+	max := c.MaxRequestsInFlight + c.MaxMutatingRequestsInFlight
```

Comment:
Hm, it's a little weird that those requests are *completely* unlimited. That seems like an oversight. We should probably limit them.

In the interest of making progress, how about we add some generous margin to this limit in this PR to account for them, and then in a follow-up set an explicit limit on long-running requests?